frontend/templates/troubleshooting/index.tsx

```diff
@@ -916,21 +929,104 @@ function RelatedProblems({ problems }: { problems: Problem[] }) {
+            </HeadingSelfLink>
+         )}
+         <Stack
             id={RelatedProblemsRecord.uniqueId}
-            pt={4}
-            onClick={onClick}
+            ref={ref}
+            className="sidebar"
+            as="aside"
+            spacing={{ base: 3, md: 8 }}
+            width={{ base: '100%' }}
+            alignSelf="start"
+            fontSize="14px"
+            minWidth="320px"
+            mt={{ base: 3, md: 0 }}
          >
-            {RelatedProblemsRecord.title}
-         </HeadingSelfLink>
-         <SimpleGrid columns={{ base: 1, sm: 2 }} gap={3} mt={4}>
-            {problems.map((problem) => (
-               <ProblemCard problem={problem} key={problem.title} />
-            ))}
-         </SimpleGrid>
-      </Box>
+            <Stack className="question" spacing={1.5}>
+               <Stack
+                  spacing={1.5}
+                  bgColor="white"
+                  border="1px solid"
+                  borderColor="gray.300"
+                  borderRadius="md"
+               >
+                  <Flex gap={2} padding={3}>
+                     <Image
+                        src={imageUrl}
+                        alt={displayTitle}
+                        boxSize={{ base: '56px', md: '96px' }}
+                        htmlWidth={96}
+                        htmlHeight={96}
+                        objectFit="cover"
+                        borderRadius="md"
+                        outline="1px solid"
+                        outlineColor="gray.300"
+                     />
+                     <Box
+                        display={{ base: 'flex', md: 'block' }}
+                        lineHeight="normal"
+                     >
+                        {/* https://github.com/iFixit/react-commerce/issues/1999 */}
+                        <Box fontWeight="semibold" my="auto">
+                           {displayTitle}
+                        </Box>
+                        {/* https://github.com/iFixit/react-commerce/issues/2000 */}
+                        {!isMobile && (
+                           <Text mt={3} noOfLines={4}>
+                              {description}
+                           </Text>
+                        )}
+                     </Box>
+                  </Flex>
+                  <Flex
+                     justifyContent="space-between"
+                     padding={3}
+                     bgColor="gray.100"
+                     borderTop="1px solid"
+                     borderColor="gray.300"
+                     borderBottomRadius="md"
+                  >
+                     {/* https://github.com/iFixit/react-commerce/issues/2001 */}
+                     {hasRelatedPages && (
+                        <Box>{problemCount} Common Problems</Box>
+                     )}
+                     {/* https://github.com/iFixit/react-commerce/issues/2002 */}
+                     <Link
+                        href={viewUrl}
```

Comment:
This fulfills requirements in https://github.com/iFixit/react-commerce/issues/2002